Stratis.Bitcoin/BlockStore/BlockStoreFeature.cs

```diff
@@ -57,13 +58,16 @@ public static class MBlockStoreBuilderExtension
 	{
 		public static IFullNodeBuilder UseBlockStore(this IFullNodeBuilder fullNodeBuilder)
 		{
-			fullNodeBuilder.ConfigureFeature(features =>
+            var blockRepository = new BlockRepository(fullNodeBuilder.Network, new DataFolder(fullNodeBuilder.NodeSettings));
```

Comment:
Yes I know that the datafolder and network are probably already registered services but I can't resolve them in the ConfigureFeature without the IServiceProvider. That would be even nicer.